src/experiments/gridworlds/JuliaRL_BasicDQN_EmptyRoom.jl

```diff
@@ -14,7 +14,7 @@ function RLCore.Experiment(
     lg = TBLogger(log_dir, min_level = Logging.Info)
     rng = StableRNG(seed)
 
-    inner_env = GridWorlds.EmptyRoom(rng = rng)
+    inner_env = GridWorlds.EmptyRoomDirected(rng = rng)
```

Comment:
Could you spit this into another PR so that the reviewing process of this PR doesn't block the CI?